auto-cert-kit.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Service to run auto cert kit test
+Requires=xapi.service
+After=xapi.service
+
+[Service]
+Type=simple
+ExecStartPre=/opt/xensource/bin/xapi-wait-init-complete 300
+ExecStart=/usr/bin/python /opt/xensource/packages/files/auto-cert-kit/test_runner.py -t /opt/xensource/packages/files/auto-cert-kit/test_run.conf
+RemainAfterExit=yes
```

Comment:
I see the definition is as below
```
If set to True, the service is considered active even when all its processes exited. Default value is False. This option is especially useful if Type=oneshot is configured.
```
Here is `yes` the same with `True`, this configuration seems answer my previous question, but will it be OK to set the service active even `ExecStart` and `ExecStartPre` existed with error?